DirectProgramming/DPC++FPGA/Tutorials/Features/fpga_reg/src/fpga_reg.cpp

```diff
@@ -0,0 +1,216 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <CL/sycl/intel/fpga_extensions.hpp>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+#include "dpc_common.hpp"
+
+using namespace sycl;
+using namespace std;
```

Comment:
These 2 `using` are considered as bad C++ practices

Reply:
While I agree that this is bad practice in principle, our coding standard for samples recommends this for simple samples to reduce overall code size.

Reply:
Indeed! "our coding standard for samples recommends this for simple samples to reduce overall code size" because even a manager without any programming experience can understand such a simple argument. :-)
Using modern C++ in another way to reduce code size otherwise...